src/bindings/cpp/include/key.hpp

```diff
@@ -134,7 +134,7 @@ class Key
 	inline ckdb::Key * operator* () const;
 
 	inline ckdb::Key * release ();
-	inline ckdb::Key * dup (elektraCopyFlags flags = KEY_CP_ALL) const;
+	inline Key dup (elektraCopyFlags flags = KEY_CP_ALL) const;
```

Comment:
Well I had to fix some SWIG bindings.
Am I sure that this *what*? 😅

Reply:
Oh... Are you sure that this doesn't cause memory issues? IIRC `dup` didn't wrap the `Key *` because that caused ref-counting issues somewhere and lead to memory leaks. Maybe that was only the case for `KeySet *` though.

In any case, shouldn't really matter since the CI would eventually complain if there are memory issues. Just wanted to tell you that this might happen